bindings/gumjs/gumscript-runtime-core.js

```diff
@@ -198,6 +198,27 @@
         }
     });
 
+    Object.defineProperty(Process, 'getModuleByName', {
+        enumerable: true,
+        value: function (name) {
+            var module = null;
+            Process.enumerateModules({
+                onMatch: function (m) {
+                    if (m.name === name) {
+                        module = m;
+                        return 'stop';
+                    }
+                },
+                onComplete: function () {
+                    if (!module) {
+                        throw new Error('Module ' + name + ' not found');
```

Comment:
It would be good to make this consistent with the other error message, so something like:
```JavaScript
"Unable to find module '" + name + "'"
```